lightwood/data/encoded_ds.py

```diff
@@ -67,10 +67,11 @@ def __getitem__(self, idx: int) -> Tuple[torch.Tensor, torch.Tensor]:
                 if hasattr(self.encoders[col], 'data_window'):
                     cols = [self.target] + [f'{self.target}_timestep_{i}'
                                             for i in range(1, self.encoders[col].data_window)]
+                    data = [self.data_frame[cols].iloc[idx].tolist()]
```

Comment:
I'm really confused by what's happening here...

Originally we do `tolist()` to convert the single element (which ends up being a pd.Series anyway) to a list to fit the signature.

In this case we need a list of lists... will the above code have `.tolist()` convert the dataframe composed of the selected columns at that index into a list ? If so that'd make sense.

Reply:
Not quite. This has to do with the following: is the expected input to encode() a list of items to encode?
If so, the list wrapper is there because of the following: say we have two rows for which we'll retrieve target_timestep_0 and target_timestep_1  (which is what this encoder does, i.e. arrays are its inputs). Then, the input to encode would be:

```
[
  [row1_time_0_value, row1_time_1_value], 
  [row2_time_0_value, row2_time_1_value], 
]
```

Which I believe conforms to the discussed interface, unless I'm off the mark here. Added types for good measure.

Reply:
yeap, all good